api/src/main/java/org/pmiops/workbench/db/WorkbenchDbConfig.java

```diff
@@ -55,4 +66,15 @@ public PlatformTransactionManager transactionManager(
       @Qualifier("entityManagerFactory") EntityManagerFactory entityManagerFactory) {
     return new JpaTransactionManager(entityManagerFactory);
   }
+
+  /**
+   * The PoolConfiguration to use for all data sources (with modification). The primary connection
```

Comment:
I've never used prototype scope before... the idea here is that you can modify PoolProperties elsewhere and it will stick?  (And if we used a singleton it wouldn't?)

Reply:
The opposite of that (I think). My understanding is that by default Spring beans will have singleton scope. Since PoolConfiguration is mutable, and we actually need to mutate it (I tried getting around that, but didn't find a way to clone), I want to instead recreate this object on each injection, which is what "prototype" scope claims to do.